introduction.md

```diff
@@ -1 +1,9 @@
+and something something
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?